src/autopredictor/fit.py

```diff
@@ -0,0 +1,25 @@
+def fit(X_train,X_test,y_train,y_test):
+
```

Comment:
I might need to double check but I believe there is no space needed in between def and the triple quote.
```suggestion
```